app/views/steps/applicant/address_details/edit.html.erb

```diff
@@ -7,7 +7,15 @@
      <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading', name: @form_object.record.full_name %></h1>
 
      <%= step_form @form_object do |f| %>
-        <%= f.text_area :address, size: '40x4', class: 'form-control-3-4' %>
+        <% if f.object.split_address? %>
```

Comment:
Take a look at https://design-system.service.gov.uk/patterns/addresses/multiple/index.html
We can try to mimic the size of the different inputs to follow the design recommendations.